openshift_scalability/install-ingress-autoscaling.sh

```diff
@@ -0,0 +1,173 @@
+#/!/bin/bash
+################################################
+## Author: qili@redhat.com
+## Description: Script to install ingress autoscaling
+## 4.12 Feature: https://issues.redhat.com/browse/NE-361
+## $1 MAXREPLICACOUNT
+## Before running this script 
+## From web console
+## 1. Operators -> OperatorHub
+## 2. Input keda in the filter and select 'Custom Metrics Autoscaler'.
+## 3. Click Install, in the opened page, click Install.
+## 4. Operators ->'Installed Operators', find 'Custom Metrics Autoscaler'.
+## 5. On 'All instances' tab, click "Create new" and select 'KedaController', click 'Create'.
+################################################
+set -e
+
+if [[ -z $1 ]]; then
+    echo "[ERROR] please provide MAXREPLICACOUNT as parameter, e.g. 1,2,omit"
+    exit 1
+fi
+keda_pod=$(oc get po -n openshift-keda | grep Running -c)
+if [[ $keda_pod -ne 3 ]]; then
+    echo "[ERROR] please make sure KedaController is successfully installed."
+    exit 1
+fi
+echo "[INFO ]====Installing Ingresss Autoscaling===="
+
+echo "[INFO] Creating project openshift-ingress-operator"
+oc project openshift-ingress-operator
+
+echo "[INFO] Applying ConfigMap cluster-monitoring-config"
+oc apply -f -<<EOF
```

Comment:
Let's add all these yaml files to the content folder or in a different folder in this directory to be reusable